runtime/kusama/constants/src/weights/block_weights.rs

```diff
@@ -42,23 +42,24 @@ use frame_support::{
 
 parameter_types! {
 	/// Time to execute an empty block.
-	/// Calculated by multiplying the *Average* with `1` and adding `0`.
+	/// Calculated by multiplying the *Average* with `1.0` and adding `0`.
 	///
 	/// Stats nanoseconds:
-	///   Min, Max: 6_094_385, 6_351_993
-	///   Average:  6_192_341
-	///   Median:   6_193_838
-	///   Std-Dev:  63893.84
+	///   Min, Max: 350_097, 393_199
+	///   Average:  356_760
+	///   Median:   353_859
+	///   Std-Dev:  8772.35
 	///
 	/// Percentiles nanoseconds:
-	///   99th: 6_332_047
-	///   95th: 6_308_225
-	///   75th: 6_236_204
-	pub const BlockExecutionWeight: Weight = WEIGHT_PER_NANOS.saturating_mul(6_192_341);
+	///   99th: 390_162
+	///   95th: 383_320
+	///   75th: 356_698
+	pub const BlockExecutionWeight: Weight = WEIGHT_PER_NANOS.saturating_mul(356_760);
```

Comment:
uhhh what happend here

Reply:
this change is consistent across runtimes, did anything change in the benchmark logic?

Reply:
I have no idea whats going on.

Reply:
aye, will do

Reply:
reverted that change